testing/google-cloud-appengine-flex-compat/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>vam-veneer</application>
+  <application>GCLOUD_PROJECT_ID_HERE</application>
```

Comment:
How does someone know to set this? When does someone set it? How do they find out the right thing to set it to?